### DISCUSSION TITLE
chatgpt web无法响应

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#50] [Q&A] [answered]
今天测试发现一个情况，使用的项目是这个https://github.com/Chanzhaoyu/chatgpt-web
验证方式是accessToken。
如果那个会话存在，就可以正常回复。
如果那个会话被我在官网clean了，就会无响应，一直在等待响应，看反代日志也没报错。如果不重启容器就无法使用了，目前这个情况是100%复现（我这里）。
![image](https://user-images.githubusercontent.com/31379266/232996374-f19231f4-b017-4731-ac95-4298f1e5ddd9.png)

**Comment by nikissXI** [comment]
我试了其他的社区反代是会返回这个提示的，大佬处理一下这个情况？
![image](https://user-images.githubusercontent.com/31379266/233000857-e503c8fd-45fb-4a5b-97b5-fe675488b26c.png)

**Reply by linweiyuan** [reply]
404 我好像没处理，晚上处理下

**Reply by nikissXI** [reply]
大佬加油！

**Reply by linweiyuan** [reply, marked as answer]
这个昨天已经加了，返回内容有点不一样

**Reply by nikissXI** [reply]
嗯嗯，昨晚拉了新的镜像了，能用！
